cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extensions.kt

```diff
@@ -535,6 +535,10 @@ val Node?.namespaces: List<NamespaceDeclaration>
 val Node?.variables: List<VariableDeclaration>
     get() = this.allChildren()
 
+/** Returns all [TupleDeclaration] children in this graph, starting with this [Node]. */
+val Node?.tuples: List<TupleDeclaration>
```

Comment:
This might be a bit misleading as 1) not all of the declarations where we use this are actually tuples and 2) one could think we mean variables of type tuple (e.g., a variable `a: Tuple`). I'm wondering if this is really relevant/would be frequently used